policy/modules/system/init.if

```diff
@@ -650,6 +650,44 @@ interface(`init_domtrans',`
 	domtrans_pattern($1, init_exec_t, init_t)
 ')
 
+########################################
+## <summary>
+##	Execute init (/sbin/init) with a domain transition
+##	to the provided domain.
+## </summary>
+## <desc>
+##	Execute init (/sbin/init) with a domain transition
+##	to the provided domain.  This is used by systemd
+##	to execute the systemd user session.
+## </desc>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to transition.
+##	</summary>
+## </param>
+## <param name="domain">
+##	<summary>
+##	New domain.
+##	</summary>
+## </param>
+#
+interface(`init_pgm_spec_user_daemon_domain',`
+	gen_require(`
+		type init_t, init_exec_t;
+	')
+
+	domain_type($1)
+	domain_entry_file($1, init_exec_t)
+
+	spec_domtrans_pattern(init_t, init_exec_t, $1)
+
+	allow init_t $1:process rlimitinh;
```

Comment:
I can buy the setsched, but what environment is being filtered such that noatsecure is required?

Reply:
Ok then for now do the setsched and just keep in the back of your mind that if you see behavior that cannot be explained, that it may be the noatsecure. I am afraid I can no longer find any justification in my git history

Reply:
Actually, i suppose you can try it out. do a systemd-run --user env with and without the noatsecure? See if theres any changes.

Reply:
i will try this see if i can refresh my memory....

Reply:
I think its would make sense though that systemd --system cleans the environment when it executes systemd --user

Reply:
I tried it out and without the noatsecure access the systemd-user instance cannot even be spawned in enforcing mode.